frontend/src/scenes/settings/organization/InviteModal.tsx

```diff
@@ -270,7 +270,7 @@ export function InviteModal({ isOpen, onClose }: { isOpen: boolean; onClose: ()
                                 <LemonButton
                                     onClick={() => inviteTeamMembers()}
                                     type="primary"
-                                    disabled={!canSubmit}
+                                    disabledReason={canSubmit ? undefined : 'Please fill out all fields'}
```

Comment:
Just making this experience better

Reply:
```suggestion
                                    disabledReason={!canSubmit ?? 'Please fill out all fields'}
```

I think you can do this too! Avoids the `undefined`

Reply:
Can't have the prop as a boolean so needs to be a str or undefined